cypress/e2e/Api.cy.js

```diff
@@ -0,0 +1,37 @@
+describe("EngageSphere API", () => {
+  const apiUrl = Cypress.env("CUSTOMERS_API_URL"); 
+  it("Verificando se na recuperação do cliente o status é 200", () => {
+    const page = 1;
+
+    cy.request({
+      method: "GET",
+      url: `${apiUrl}?page=${page}`, 
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+    });
+  });
+
+  it("Recuperando 10 clientes com sucesso", () => {
```

Comment:
Uma sugestão melhor para a descrição do caso de testes seria:

```suggestion
  it("retorna 10 clientes quando limit é iqual a 10", () => {
```